docs/adr/006-start-off-with-server-side-rendering.md

```diff
@@ -0,0 +1,36 @@
+# 6. Front-end tech stack: Start off with server-side rendering
+
+Date: 2021-07-14
+
+## Status
+
+Proposed
```

Comment:
This should be Accepted before we merge.

Reply:
Good catch!